CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Added
+- New `Reset` flow which can be triggered manually for `Root` and `Derived` datasets
```

Comment:
Please indicate that this applies to GraphQL (CHANGELOG.md reader, user, ideally should immediately understand how to use the new added functionality OR understand where to go to find out more details)